openapi/resource-server.yaml

```
@@ -200,6 +201,10 @@ paths:
                 externalRef:
                   type: string
                   description: A reference that can be used by external systems to reconcile this payment with their systems. E.g. An invoice number. (Optional)
+                metadata:
+                  type: string
+                  format: json
+                  description: Additional metadata associated with the incoming payment, represented as a JSON string. (Optional)
```

```suggestion
                metadata:
                  type: string
                  format: json
                  description: Additional metadata associated with the incoming payment, represented as a string. (Optional)
```
We can have the JSON string in (some?all?) of the examples, but any string is valid

---

That is a good point. I was assuming it should be an object so I was including that note to show that intention since the route will accept any string. Guess I should remove "represented as a JSON string" regardless of what we decide about keeping as string vs. changing to object.